contrib/forms/body_composition/new.php

```diff
@@ -68,40 +68,25 @@ function rbinput($name, $value, $desc, $colname)
  // If updating an existing form...
  //
     if ($formid) {
-        $query = "UPDATE form_body_composition SET "      .
-         "body_type = "             . rbvalue('form_body_type') . ", "  .
-         "height = '"               . form2db($_POST['form_height'])     . "', " .
-         "weight = '"               . form2db($_POST['form_weight'])     . "', " .
-         "bmi = '"                  . form2db($_POST['form_bmi'])        . "', " .
-         "bmr = '"                  . form2db($_POST['form_bmr'])        . "', " .
-         "impedance = '"            . form2db($_POST['form_impedance'])  . "', " .
-         "fat_pct = '"              . form2db($_POST['form_fat_pct'])    . "', " .
-         "fat_mass = '"             . form2db($_POST['form_fat_mass'])   . "', " .
-         "ffm = '"                  . form2db($_POST['form_ffm'])        . "', " .
-         "tbw = '"                  . form2db($_POST['form_tbw'])        . "', " .
-         "other = '"                . form2db($_POST['form_other'])      . "' "  .
-         "WHERE id = '$formid'";
-        sqlStatement($query);
+        $query = "UPDATE form_body_composition SET 
+        body_type = ?, height = ?, weight = ?, bmi = ?, bmr = ?, impedance = ?, 
+        fat_pct = ?, fat_mass = ?, ffm = ?, tbw = ?, other = ? WHERE id = ?";
+         
+        sqlStatement($query, array(rbvalue('form_body_type'),  form2db($_POST['form_height']), form2db($_POST['form_weight']), form2db($_POST['form_bmi']), 
```

Comment:
Lets fixup the rbvalue function to work with binding. In that function:
Change:
```
return "'$tmp'";
```
to:
```
return "$tmp";
```
(note I just removed the single quotes)